sources/packages/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Get, Injectable, Param } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import BaseController from "../BaseController";
+import { StudentUploadFileDTO } from "./models/student.dto";
+import { StudentControllerService } from "./student.controller.service";
+
+/**
+ * Student controller for AEST Client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("students")
+@ApiTags("students")
+@Injectable()
+export class StudentAESTController extends BaseController {
+  constructor(
+    private readonly studentControllerService: StudentControllerService,
+  ) {
+    super();
+  }
+
+  /**
+   * This controller returns all student documents uploaded
+   * by student uploader
+   * @returns list of student documents as StudentUploadFileDTO
+   */
+  @Get("documents/:studentId")
```

Comment:
I know that it was not really introduced in this PR but just a thought, is it `"documents/:studentId"` or ":studentId/documents"? 🤔

Reply:
he he, I feel we should have ":studentId/documents" as the overall route will look like /api/aest/students/documents/326
 /api/students/students/documents 

instead 

/api/aest/students/326/documents
/api/students/students/documents

This will be the difference, I can change it. Thanks for the suggestion

Reply:
For students, `studentId` is used from the token. here I guess, it should `":studentId/documents"`

Reply:
I believe it should be ("documents/student/:studentId")